app/pages/lab-fem/workflow.jsx

```diff
@@ -118,6 +119,11 @@ class EditWorkflowPage extends Component {
     return this.setState(prevState => ({ showTaskAddButtons: !prevState.showTaskAddButtons }));
   }
 
+  toggleCaesarEnabled(e) {
+    return this.props.workflow.update({
+      'configuration.enable_caesar_fetching': e.target.checked
```

Comment:
Double checking: https://github.com/zooniverse/front-end-monorepo/pull/6039 assumes `enable_caesar_data_fetching` but here the configuration key is `enable_caesar_fetching` -- which are you planning to adopt?

Reply:
I had to fix an errant file getting added to the commit that shouldn't been in there and didn't push up the appropriate change. This should be fixed and matching FEM.